library/core/functions.general.php

```diff
@@ -746,6 +734,18 @@ function dbdecode($value) {
     }
 }
 
+if (!function_exists('dbencode')) {
+    /**
+     * Encode a value in preparation for database storage.
+     *
+     * @param mixed $value
+     * @return string
+     */
+    function dbencode($value) {
+        return Gdn_Format::serialize($value);
```

Comment:
`dbencode` and `dbdecode` are not meant to be drop-in replacements for `serialize` and `unserialize`.  They're only meant to be helpers for getting complex data into and out of the database.  I don't believe there is currently a legitimate case wherein a string or primitive type needs to undergo encoding, such a serialization, to be stored.  To my knowledge, the types of data needing to be encoded are exclusively arrays and objects.  If we're trying to serialize strings and primitives somewhere, I think that's a separate issue that should be addressed.